utils/pretty.py

```diff
@@ -24,13 +24,55 @@ def x(o):
     return x
 
 
+def process_attrs(attrs):
+    """Enables us for more freedom in attribute specification"""
+    if isinstance(attrs, basestring):
+        return [attrs]
+    elif attrs is None:
+        return []
+    elif isinstance(attrs, (list, tuple)):
+        return attrs
+    else:
+        raise TypeError(
+            "pretty_attrs should be tuple, list, str or None, not {}".format(type(attrs).__name__))
+
+
 class Pretty(object):
-    """A mixin that prints repr as <MyClass field1=..., field2=...>. The
-       fields that will be printed should be stored in the class's
-       pretty_attrs attribute (none by default).
+    """A mixin that prints repr as <MyClass field1=..., field2=...>. Provides default constructor
 
+    The fields that will be printed should be stored in the class's pretty_attrs attribute
+    (none by default). It also provides a default constructor for the values specified.
+
+    the ``pretty_attrs`` can be list, tuple, str or None. The ``required_attrs`` is a list of
```

Comment:
I meant it for being in the same style as writing tuples without the parentheses, eg.:
```python
a = 1, 2, 3  # -> tuple
```
So that when it sees a string, it assumes it is a 1-iterable.